alchemy/src/vercel/project.ts

```diff
@@ -0,0 +1,418 @@
+     */
+    repo: string;
+  };
+
+  /**
+   * The install command for this project. When `null` is used this value will be automatically detected
+   */
+  installCommand?: string;
+
+  /**
+   * The output directory of the build. When `null` is used this value will be automatically detected
+   */
+  outputDirectory?: string;
+
+  /**
+   * Specifies whether the source code and logs of the deployments for this project should be public or not
+   */
+  publicSource?: boolean;
+
+  /**
+   * The name of a directory or relative path to the source code of your project. When `null` is used it will default to the project root
+   */
+  rootDirectory?: string;
+
+  /**
+   * The region to deploy Serverless Functions in this project
+   */
+  serverlessFunctionRegion?: string;
+
+  /**
+   * Specifies whether Zero Config Failover is enabled for this project.
+   */
+  serverlessFunctionZeroConfigFailover?: boolean;
+
+  /**
+   * OpenID Connect JSON Web Token generation configuration.
+   */
+  oidcTokenConfig?: {
+    /**
+     * Whether or not to generate OpenID Connect JSON Web Tokens.
+     */
+    enabled: boolean;
+
+    /**
+     * team: `https://oidc.vercel.com/[team_slug]` global: `https://oidc.vercel.com`
+     */
+    issuerMode: "team" | "global";
+  };
+
+  /**
+   * Opt-in to skip deployments when there are no changes to the root directory and its dependencies
+   */
+  enableAffectedProjectsDeployments?: boolean;
+
+  /**
+   * Resource configuration
+   */
+  resourceConfig?: {
+    /**
+     * Whether fluid is enabled
+     */
+    fluid?: boolean;
+
+    /**
+     * Default regions for functions
+     */
+    functionDefaultRegions?: string[];
+
+    /**
+     * Default timeout for functions
+     */
+    functionDefaultTimeout?: number;
+
+    /**
+     * Default memory type for functions
+     */
+    functionDefaultMemoryType?: "standard_legacy" | "standard" | "performance";
+
+    /**
+     * Whether function zero config failover is enabled
+     */
+    functionZeroConfigFailover?: boolean;
+
+    /**
+     * Whether elastic concurrency is enabled
+     */
+    elasticConcurrencyEnabled?: boolean;
+
+    /**
+     * The build machine type
+     */
+    buildMachineType?: "enhanced" | "ultra";
+  };
+}
+
+/**
+ * Output returned after Project creation/update
+ */
+export interface Project extends Resource<"vercel::Project">, ProjectProps {
+  /**
+   * The ID of the project
+   */
+  id: string;
+
+  /**
+   * The account ID that the project belongs to
+   */
+  accountId: string;
+
+  /**
+   * The time at which the project was created
+   */
+  createdAt: number;
+
+  /**
+   * The time at which the project was last updated
+   */
+  updatedAt: number;
+
+  /**
+   * The latest deployment of the project
+   */
+  latestDeployment?: {
+    /**
+     * The ID of the deployment
+     */
+    id: string;
+
+    /**
+     * The URL of the deployment
+     */
+    url: string;
+  };
+}
+
+export const Project = Resource(
```

Comment:
docs and examples

Reply:
Yessir 🫡

Reply:
Resolved via 5e341c7c8df7a0b072150e4afe04f8430681a36b